typings/balena-sdk.d.ts

```diff
@@ -571,26 +577,30 @@ declare namespace BalenaSdk {
 		release: NavigationResource<Release>;
 	}
 
+	// See: https://github.com/balena-io/resin-proxy/issues/51#issuecomment-274251469
+	interface OsUpdateActionResult {
+		status: 'idle' | 'in_progress' | 'update_done' | 'error';
```

Comment:
These are changing/have changed so it'd be good to stick with the newer form (or accept both), cc @xginn8

Reply:
I saw the UI PR after I opened this one.
Will update the typings here, thanks for the reminder :+1: